conda-arm64-requirements.txt

```diff
@@ -1,4 +1,4 @@
 # Required for v3io-frames client through grpcio-tools:
 # https://github.com/v3io/frames/blob/5490b1e7f281a8c44ec679105957e181c3fa81fc/clients/py/requirements.txt#L4
 # grpcio-tools 1.48.2 requires protobuf<4.0dev,>=3.12.0
-protobuf>=3.20.3, <4
+protobuf>=3.20.3, <4; python_version < "3.11"
```

Comment:
With this change:
```
$ make install-conda-requirements
conda install --yes --file conda-arm64-requirements.txt

InvalidMatchSpec: Invalid spec 'protobuf >=3.20.3,<4;python_version<"3.11"': Invalid version '4;python_version<"3.11"': invalid character(s)

make: *** [install-conda-requirements] Error 1
```
Conda has a different spec, see:
https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/pkg-specs.html#package-match-specifications

Reply:
ill undo the change for now. will update when py311 is fully supported